Foundation/Stream.swift

```diff
@@ -119,7 +119,7 @@ open class InputStream: Stream {
 
     // reads up to length bytes into the supplied buffer, which must be at least of size len. Returns the actual number of bytes read.
     open func read(_ buffer: UnsafeMutablePointer<UInt8>, maxLength len: Int) -> Int {
-        return CFReadStreamRead(_stream, buffer, CFIndex(len._bridgeToObjectiveC()))
+        return CFReadStreamRead(_stream, buffer, CFIndex(truncating: len._bridgeToObjectiveC()))
```

Comment:
This is produced by the Xcode fixit. Could it just be `return CFReadStreamRead(_stream, buffer, len)` though?

Reply:
If `len` is an `Int` and `CFIndex` is defined as `public typealias CFIndex = Int` why is there any bridging at all?

Reply:
That's what I wondered!

Reply:
Yea let's drop the extra round trip bridge here